subscriptions/function/cu/02_cu_subscription.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps.open-cluster-management.io/v1
+kind: Subscription
+metadata:
+  name: cu-subscription
+  namespace: cu-subscriptions
+  labels:
+    app: cu-subscription
+  annotations:
+    apps.open-cluster-management.io/git-path: manifests/function/cu
```

Comment:
that doesnt exist yet
and create a 'soft' link between the repos
which we should re-consider